be/src/common/config.cpp

```diff
@@ -131,6 +131,8 @@ DEFINE_mBool(disable_memory_gc, "false");
 
 DEFINE_mInt64(large_memory_check_bytes, "2147483648");
 
+DEFINE_mBool(disable_memory_orphan_check, "false");
```

Comment:
Better to use enable_memory_orphan_check

Reply:
fixed